examples/Folders/Readme.md

```diff
@@ -0,0 +1,45 @@
+Dropzone that accepts folders as drag-and-drop. Supports multiple folders and subfolders.
```

Comment:
I tested the example and it works great. But the documentation still missing bits on the plugin architecture IMO. I would add more information about the architecture and emphasize the pluggable part. We would need to show function signature and an isolated example with promises IMO.

Reply:
Do you mean that we need to explain what the plugin does(how it makes folder processing possible) and how the returned value(list of files in the folder) is then used in Dropzone component utilizing promises? I may think not that deep but in the light of my comment regarding naming maybe we could overload this component users little bit less with how it works and emphasize more what it solves. But anyways I think both variants are possible and docs can also contain info on the architecture.
If you want me to add it I think I need a little explanation of what you mean by function signature and isolated example though

Reply:
@d-ivashchuk no I mean the API part. Function signature etc. Your example is just one case of what's possible so we need to add this to the general docs (ideally this would live inside the component using JSDoc and styleguidist will extract this) but not to this example.